rero_ils/modules/templates/jsonschemas/templates/template-v0.0.1.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Template",
+  "description": "JSON schema for templates.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "template_type",
+    "name",
+    "description",
+    "creator",
+    "organisation",
+    "visibility"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate templates records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schemas/templates/template-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Template ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "name": {
+      "title": "Template name",
+      "description": "The name of the template.",
+      "type": "string",
+      "minLength": 2
+    },
+    "description": {
+      "title": "Template description",
+      "description": "The description of the template.",
+      "type": "string"
+    },
+    "organisation": {
+      "title": "Organisation",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Organisation URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/organisations/.*?$"
+        }
+      }
+    },
+    "creator": {
+      "title": "Creator",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Creator URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/patrons/.*?$"
+        }
+      }
+    },
+    "visibility": {
+      "type": "string",
+      "title": "Visibility",
```

Comment:
Use always the same order of proprieties: title, description (if any), type, ...
https://github.com/rero/developer-resources/blob/master/data/jsonschema.md#order